docs/userGuide/siteConfiguration.md

```diff
@@ -118,4 +118,8 @@ The ignore pattern follows the [glob pattern used in .gitignore](https://git-scm
 
 **The level of headings to be indexed for searching.** Default: `3` %%i.e., only headings of levels 1,2,3 will be indexed for searching%%.
 
+#### **`disableSearch`**
+
+**If set to `true`, MarkBind will not index pages for searching.** See [Use Searchbar](usingComponents.html#search-bar) for more information.

```

Comment:
```
**Specifies that the website should use MarkBind's search functionality.** See [User Guide: Making the Site Searchable](userguide/makingthesitesearchable) for more details.
```

Reply:
Should this be true by default?

Reply:
Yes it should be true by default for `enableSearch`.